docs(common): remove mention of the deprecated web worker platform. Minor grammar/stylistic changes.

LayZeeDK opened the pull request:
Remove mention of the deprecated web worker platform.

Minor grammar/stylistic changes.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
Inline documentation for the `PlatformLocation` service mentions the deprecated `@angular/platform-webworker`.

A few typos.

Issue Number: N/A


## What is the new behavior?
Inline documentation for the `PlatformLocation` service instead mentions `@angular/platform-server`.

Typos corrected, minor grammar and stylistic changes.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

AndrewKushnir reviewed (approved):
Hi @LayZeeDK, thanks for creating this PR 👍 

The content looks good from eng. perspective, but I've added @kapunahelewong as a reviewer to make sure it's aligned with docs requirements/style.

FYI, the `lint` CircleCI job is failing due to the missing commit message body (which should have at least 100 chars), so I've added the "cleanup" label. Could you please update commit message and include the content of the `What is the new behavior` section as a commit message body?

Thank you.

LayZeeDK committed:
docs(common): remove mention of the deprecated
web worker platform. Minor grammar/stylistic changes.

Inline documentation for the PlatformLocation service instead mentions @angular/platform-server.

Typos corrected, minor grammar and stylistic changes.

kapunahelewong reviewed (approved):
Thanks, @LayZeeDK!!

Reviewed-for: global-docs-approvers
